compiler/rustc_span/src/analyze_source_file.rs

```diff
@@ -185,6 +186,165 @@ cfg_match! {
         }
     }
 }
+
+#[cfg(not(bootstrap))]
+cfg_match! {
```

Comment:
Please don't duplicate such large amounts of complicated code next time, it causes headaches when someone is trying to change it (which happened here). Instead it would have been nicer if you had reverted this back to use `cfg_if` or just plain cfg until the bootstrap compiler had the new `cfg_match` syntax.